lib/ChainFind.js

```diff
@@ -223,20 +223,6 @@ function ChainFind(Model, opts) {
       chainRun(cb);
       return this;
     },
-    success: function (cb) {
-      if (!promise) {
-        promise = new Promise();
-        promise.handle(this.all);
-      }
-      return promise.success(cb);
-    },
-    fail: function (cb) {
-      if (!promise) {
-        promise = new Promise();
-        promise.handle(this.all);
-      }
-      return promise.fail(cb);
-    },
```

Comment:
I recommend we keep these in the code, and just add a `console.warn` deprecation message & mention deprecation in changelog